redis/Cargo.toml

```diff
@@ -131,6 +130,7 @@ socket2 = "0.4"
 assert_approx_eq = "1.0"
 fnv = "1.0.5"
 futures = "0.3"
+futures-time = "3"
```

Comment:
let's remove it entirely

Reply:
How do you want to do this? `test_block_on_all_panics_from_spawns()` seems to already rely on tokio unconditionally (via the use of `tokio::task::spawn()`), is it okay to do that for the `sleep()` calls, too? `Runtime` isn't a part of the public API (and I'm honestly not motivated to spend a lot of time on cleaning up the tests, since the dev-dependencies don't bother me).

Reply:
fair enough, not critical.